### PR TITLE
Add CLI

### PR DESCRIPTION
**Pull request by marcussacana**
Command lines are useful :3

### CODE REVIEW

File: ReadMe.md [hunk 1]
```diff
@@ -19,6 +19,13 @@ A simple tool to decompress `.zs` files in Totk, compression support is also pla
 - Browse for a `.zs` file in your Totk game dump
 - Click `Decompress` and save the raw file to your computer
 
+## Usage (CLI)
+- For CLI works, you must setup the game path before.
```
Comment (ArchLeaders) [inline ReadMe.md]:
```suggestion
- To use the CLI tools you must first setup the game paths in the user-interface.
```

File: ReadMe.md [hunk 2]
```diff
@@ -19,6 +19,13 @@ A simple tool to decompress `.zs` files in Totk, compression support is also pla
...
+- Decompress `Totk.ZStdTool.exe -x Something.zs`
+  - The output will be in the same folder but without `.zs` extension
+- Compress `Totk.ZStdTool.exe -c Something.bin`
+  - The output will be in the same folder but with `.zs.new` extension
```
Comment (ArchLeaders) [inline ReadMe.md]: - An output flag (`-o/--output`) should be provided for both of these commands
- Only `.zs` should be appended to the file name, adding `.new` is confusing and unnecessary

File: src/Program.cs [hunk 3]
```diff
@@ -9,8 +11,62 @@ internal class Program
...
+        {
+            CLI(args);
+        }
+        else
+        {
+            BuildAvaloniaApp().StartWithClassicDesktopLifetime(args);
+        }
+    }
+
+    private static void CLI(string[] Args)
```
Comment (ArchLeaders) [inline src/Program.cs]: Unclean function name, this looks more like a class name. Something like `ProcessArgs(string[] args)` might be clearer.

File: src/Program.cs [hunk 4]
```diff
@@ -9,8 +11,62 @@ internal class Program
...
+    {
+        DllManager.LoadCead();
+
+        bool CompressMode = false;
+
+        while (Args.Any())
+        {
+            var Entry = Args.First();
+            Args = Args.Skip(1).ToArray();
```
Comment (ArchLeaders) [inline src/Program.cs]: Args should be passed in as a `List<string>` to avoid copying the array every iteration

**Review by ArchLeaders — CHANGES_REQUESTED** [review]
General formatting should be fixed to conform with the rest of the codebase
- Camel case arguments and variables
- Strongly typed variables (not using `var`)
- Inline curly braces (will be done automatically by my IDE, so don't worry about this one)

File: src/Program.cs (hunk 3 above)
Comment (ArchLeaders) [reply src/Program.cs]: Not sure why it picked up lines `23-25`, just ignore those

File: ReadMe.md (hunk 2 above)
Comment (marcussacana) [reply ReadMe.md]: .zs.new was intended to prevent accidental overwrite, but with -o should not be a problem

**Comment by marcussacana** [comment]
What about like this?

**Comment by ArchLeaders** [comment]
That should be good, I'll just clean up the variable names and then merge it in. Thanks for the contribution.

**Pull request merged**